### DISCUSSION TITLE
This version of ChromeDriver only supports Chrome version 113 Current browser version is 115.0.5790.110

### DISCUSSION THREAD

**Discussion by kojakeugenio** [ultrafunkamsterdam/undetected-chromedriver#1429] [General]
How to fix this error?

`Exception has occurred: SessionNotCreatedException
Message: session not created: This version of ChromeDriver only supports Chrome version 113
Current browser version is 115.0.5790.110 with binary path C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
Stacktrace:`

**Comment by emilythorne1997** [comment]
Same today, maybe we should downgrade the chrome version before the new version of undetected release

**Reply by emilythorne1997** [reply]
Also im stuck in with downgrade step to 113 chrome version, any one can help?  Thanks!

**Reply by jeffliaogithub** [reply]
Uninstall chrome
Download version 113
Install
Disable google update service

**Reply by vahner** [reply]
Search google offline installer version 113.
Also read how to disable automatic updates.
Unfortunately, I can not provide you with links, since they are in Russian

**Reply by jeffliaogithub** [reply]
https://www.browserstack.com/guide/downgrade-to-older-versions-of-chrome

**Reply by emilythorne1997** [reply]
its modified with russian lang so i dont trust that version, have tried that.

**Reply by jeffliaogithub** [reply]
I just tried today and worked for me

**Reply by kojakeugenio** [reply]
Will the chrome profiles I created get loss when I uninstall chrome?

**Reply by brrrada** [reply]
Same issue. 

Some searches online involved: installing webdriver-manager and downgrading selenium to 4.9. No results. Any other options?

**Comment by brrrada** [comment]
There's an PR open for this issue. Let's wait for the merge. https://github.com/ultrafunkamsterdam/undetected-chromedriver/pull/1427

**Reply by kojakeugenio** [reply]
I saw your comment on that thread. Did you fix yours?

**Reply by brrrada** [reply]
Ive got it to work with a downgrade to selenium 4.9

**Comment by kojakeugenio** [comment]
I manage to make it work by using portable chrome. Follow the steps below
https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/641

**Comment by mtomassoli** [comment]
There's no need to downgrade. Judging from the code, `uc` doesn't refresh `undetected_chromedriver.exe` even when it should. The solution, which I could only test on my own system, is to just delete it. You can do it by hand or by using the following code:

```python
from undetected_chromedriver.patcher import Patcher
from pathlib import Path

uc_driver_path = Patcher().executable_path
print(uc_driver_path)
if uc_driver_path.endswith('undetected_chromedriver.exe'):      # to be safe
    Path(uc_driver_path).unlink()
```

Basically, `Patcher` is responsible for downloading the original `chromedriver.exe` and patching it, resulting in the patched version called `undetected_chromedriver.exe`. You may want to use this code just once, hoping that it won't be needed anymore in the future. A better way would be to either fix `uc` or use `Patcher`'s methods to keep the exe always up-to-date.

**Comment by aiellog** [comment]
Do we know if this will be fixed automatically with the next update / push or will we have to modify our code?

**Comment by dragonsxslayer** [comment]
this workis  : pip install git+https://www.github.com/ultrafunkamsterdam/undetected-chromedriver@master
